master/static/srv/proto_get_trial.sql

```diff
@@ -0,0 +1,8 @@
+SELECT t.id,
+  t.experiment_id,
+  'STATE_' || t.state AS state,
+  t.start_time,
+  t.end_time,
+  t.hparams
```

Comment:
Should we include `total_batches_processed` here?

Reply:
+1